priv/rest.erl.eex

```diff
@@ -23,7 +23,8 @@
 
 <%= action.function_name %>(Client<%= AWS.CodeGen.RestService.required_function_parameters(action) %>, QueryMap, HeadersMap, Options0)
   when is_map(Client), is_map(QueryMap), is_map(HeadersMap), is_list(Options0) ->
-    Path = ["<%= AWS.CodeGen.RestService.Action.url_path(action) %>"],
+    Path = ["<%= AWS.CodeGen.RestService.Action.url_path(action) %>"],<%= if context.endpoint_prefix == "s3" and context.endpoint_prefix != "s3-control" do %>
```

Comment:
I think this conditional check could go in a separated function (maybe in a helper module). WDYT?